### DISCUSSION TITLE
Install of OctoEverywhere

### DISCUSSION THREAD

**Discussion by Floris039** [Guilouz/Creality-Helper-Script-Wiki#514] [Issues]
When I try to install OctoEveryhwere the scrips runs into an error. It happens after it's trying to do: "Installing or updating required python libs..."

I get the error: ERROR: Could not install packages due to an OSError: [Errno 28] No space left on device.

Can anyone please help me. Thanks in advance!

**Comment by Guilouz** [comment]
Your answer is in the error message: No space left on device.
